src/utilities/zk_pdl_with_slack/mod.rs

```diff
@@ -46,7 +46,6 @@ pub struct PDLwSlackStatement {
 pub struct PDLwSlackWitness {
     pub x: FE,
     pub r: BigInt,
```

Comment:
good catch.  Need to be changed in the description (line 22) as well.

Reply:
Updated to doc